SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -12,12 +12,14 @@ type Void {
 
 component UnrealClientRPCEndpoint {
     id = 9990;
+    bool ready = 1;
     event UnrealRPCPayload unreliable_client_to_server_rpc;
```

Comment:
I would prefer these names to change but this can wait till we decide what should these components actually look like.

Reply:
As these are transient I actually think renaming them might be free.

Reply:
Would dropping the `unreliable` part make it a good name? `unreliable_client_to_server_rpc` -> `client_to_server_rpc` (same for Server)? @mattyoung-improbable is it safe to make this change now or will it break something?

Reply:
I'd say that client_to_server_rpc is too ambiguous and they should be called rpc_event_sender or something. These are channels we can use rather than high level APIs. Although just dropping the worker unreliable would still be better.
The issue I was worried about yesterday was mostly about changing fields, events and command are easier.
However I'm not sure how we deal with loading snapshots after changing just the name of a transient message in schema.

Reply:
I'd rather keep the words `client` and `server` in variable names. In my opinion, it makes the event direction more explicit.

Reply:
That is already encoded in the component name.

Reply:
If there are no strong objections - I would prefer to keep client_to_server_rpc and server_to_client_rpc.

Reply:
Renamed `unreliable_client_to_server_rpc` -> `client_to_server_rpc_event`, `reliable_server_to_client_rpc` -> `server_to_client_rpc_command` for Client Endpoint (and correspondingly for the Server Endpoint)